src/i18n/i18n.ts

```diff
@@ -3,8 +3,14 @@ import { initReactI18next } from 'react-i18next'
 import Backend from 'i18next-http-backend'
 import LanguageDetector from 'i18next-browser-languagedetector'
 
-i18n.use(Backend).use(LanguageDetector).use(initReactI18next).init({
-  fallbackLng: 'en',
-})
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    debug: true,
```

Comment:
This will show i18n in debug mode in the dev console for the app, open to removing it but it shouldn't be noisy.